src/app/factories/google-analytics.factory.ts

```diff
@@ -0,0 +1,42 @@
+import { RequestHandler } from 'express'
```

Comment:
instead of artificially separating this from the frontend module, should we just move it into that module instead?